usr/share/rear/conf/default.conf

```diff
@@ -918,9 +918,27 @@ BACKUP_PROG_SUFFIX=".tar"
 # http://git.savannah.gnu.org/cgit/tar.git/plain/NEWS?id=release_1_27
 BACKUP_PROG_COMPRESS_OPTIONS=( --gzip )
 BACKUP_PROG_COMPRESS_SUFFIX=".gz"
-# Addons for encryption of the backup (currently only tar is supported)
-BACKUP_PROG_CRYPT_ENABLED=0
-BACKUP_PROG_CRYPT_KEY=""
+# Addons for encryption and decryption of the backup (currently only tar is supported):
+BACKUP_PROG_CRYPT_ENABLED="false"
+# When BACKUP_PROG_CRYPT_ENABLED is set to a true value, BACKUP_PROG_CRYPT_KEY must be also set.
+# There is no BACKUP_PROG_CRYPT_KEY value in etc/rear/local.conf in the ReaR recovery system.
+# It gets removed by build/default/960_remove_encryption_keys.sh
+# because the ReaR recovery system must be free of secrets
+# cf. the reasoning about SSH_UNPROTECTED_PRIVATE_KEYS below
+# and see https://github.com/rear/rear/issues/2155
+# Therefore BACKUP_PROG_CRYPT_KEY must be manually set before running "rear recover".
+# BACKUP_PROG_CRYPT_KEY is set to a default value here only
+# if not already set so that the user can set it also like
+#   export BACKUP_PROG_CRYPT_KEY="my_secret_key"
```

Comment:
@jsmeix Good point, perhaps, we should also add it to https://github.com/rear/rear/blob/master/doc/user-guide/04-scenarios.adoc ?

Reply:
done